tmail-backend/tmail-third-party/ai-bot/src/main/java/com/linagora/tmail/mailet/rag/RagListener.java

```diff
@@ -123,10 +129,19 @@ public Publisher<Void> reactiveEvent(Event event) {
                         }
                         LOGGER.info("RAG Listener triggered for mailbox: {}", addedEvent.getMailboxId());
                         MailboxSession session = mailboxManager.createSystemSession(addedEvent.getUsername());
+                        OpenRagHttpClient ragondinHttpClient = new OpenRagHttpClient(ragConfig);
                         return Mono.from(messageIdManager.getMessagesReactive(addedEvent.getMessageIds(), FetchGroup.FULL_CONTENT, session))
-                            .doOnError(error -> LOGGER.error("Error occurred: ", error.getMessage()))
-                            .flatMap(this::asRagLearnableContent)
-                            .doOnNext(text -> LOGGER.info("RAG Listener successfully processed mailContent ***** \n{}\n *****", text))
+                            .doOnError(error -> LOGGER.error("Error occurred: {}", error.getMessage()))
```

Comment:
I think I aalready commented that line that it's terrible error handling and that I wanted it removed.